front/app/containers/Admin/formBuilder/components/FormFields/index.tsx

```diff
@@ -88,6 +95,43 @@ const FormFields = ({
     }
   `;
 
+  const getTitleFromAnswerId = (
```

Comment:
Minor: We could move some these util functions to a util file. This would make this component smaller and easier to read down the line. Similar to what you did for the others.